src/main/java/org/kuali/kfs/kim/service/impl/UiDocumentServiceImpl.java

```diff
@@ -771,8 +771,8 @@ protected List<RoleMember> populateRoleMembers(
                         if (roleMember.getActiveFromDate() != null) {
                             roleMemberImpl.setActiveFromDateValue(
                                     new java.sql.Timestamp(roleMember.getActiveFromDate().getTime()));
-                        } else {
-                            // CU Customization KFSPTS-27162 Set default for activeFromDate field
+                        } else if(!roleMember.isEdit()){
```

Comment:
Missing whitespace before and after the parenthesized condition.

Reply:
updated